src/main/java/org/jenkinsci/plugins/workflow/steps/StepContext.java

```diff
@@ -149,4 +149,9 @@ public boolean hasBody() {
     public abstract int hashCode();
 
     private static final long serialVersionUID = 1L;
+
+    /**
+     * Returns the function name used in pipeline Jenkinsfile for the current StepNode.
+     */
+    public abstract String getStepDisplayFunctionName() throws IOException;
```

Comment:
Do we really need this? I think callers could already use `.get(FlowNode.class).getDisplayFunctionName()`, which is handled directly by `DefaultStepContext` as of https://github.com/jenkinsci/workflow-support-plugin/pull/252, and this still throws an exception, so it's not really much easier to use. Maybe I am missing something though.

Reply:
`FlowNode` is not accessible from this plugin, hence my idea of exposing `StepDescriptor` https://github.com/jenkinsci/workflow-step-api-plugin/pull/204#discussion_r1965996465

Reply:
fixed as in the above comment. ty